src/segment-loader.js

```diff
@@ -26,6 +26,10 @@ import shallowEqual from './util/shallow-equal.js';
 // in ms
 const CHECK_BUFFER_DELAY = 500;
 const finite = (num) => typeof num === 'number' && isFinite(num);
+// With most content hovering around 30fps, if a segment has a duration less than
+// one frame, the bandwidth and throughput calculations will not accurately reflect
+// the rest of the content.
+const MIN_SEGMENT_DURATION_TO_SAVE_STATS = 1 / 30;
```

Comment:
would it make sense to use `FRAME-RATE` from the playlist if it exists here so that this doesn't mess with calculations for 60fps content? Could be a future optimization too I guess.

Reply:
Maybe this should just be 1 / 60 instead? Not sure if that would be too low though.

Reply:
I figured even for 60 fps content we'd probably be safe (it's possible, though I'd imagine unlikely, for someone to send less than 2 frames of content, even at 60fps), but it could be worth noting here. 1 / 60 may be cutting it close, as it should work when rounded, but might be problematic if truncated or floored.

Reply:
Thinking about it again though, maybe it is worth it to just use 1/60. Will give it some testing.